services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+from django.db.models import QuerySet
+
+
+def create_movie_session(
+    movie_show_time: int,
+    movie_id: int,
+    cinema_hall_id: int,
+) -> QuerySet:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if session_date:
+        return MovieSession.objects.all().filter(show_time__date=session_date)
+    return MovieSession.objects.all()
```

Comment:
```suggestion
    movie_sessions = MovieSession.objects.all()
    if session_date:
        return movie_sessions.filter(show_time__date=session_date)
    return movie_sessions
```
No need to duplicate code, this can help support your code in future